bambu-farm-client/cpp/bambu_networking.hpp

```diff
@@ -84,8 +84,7 @@ namespace BBL
 
 #define BAMBU_NETWORK_LIBRARY "bambu_networking"
 #define BAMBU_NETWORK_AGENT_NAME "bambu_network_agent"
-#define BAMBU_NETWORK_AGENT_VERSION "01.07.04.02"
-
+#define BAMBU_NETWORK_AGENT_VERSION "01.07.07.89"
```

Comment:
Can you document in a comment here which version of Bambu Studio this works with? Could possibly be multiple, but as long as we get one working version that'll make it easier for people to use.

Reply:
It works with Bambu Studio 1.7.7.89, I added that as a comment in the code and squashed the commits in this PR.